cloudinit/sources/DataSourceAliYun.py

```diff
@@ -61,12 +69,38 @@ def parse_public_keys(public_keys):
     return keys
 
 
+class DataSourceAliYunLocal(DataSourceAliYun):
```

Comment:
Is the `get_data()` implementation in this class needed? It doesn't seem to be doing anything.

I think this could be simplified to:
```python
class DataSourceAliYunLocal(DataSourceAliYun):
    perform_dhcp_setup = True
```

Reply:
yes, you are right . I will change it .